dist/uci-console/bundles/samagra-x-uci-console.umd.js

```diff
@@ -6867,57 +2657,49 @@
+        LibEntryComponent.ctorParameters = function () { return [
+            { type: router.ActivatedRoute },
+            { type: UciService },
+            { type: ConfigService },
+            { type: common.Location },
+            { type: UciEventsService },
+            { type: TelemetryUtilsService },
+            { type: GlobalService },
+            { type: router.Router }
+        ]; };
+        __decorate([
+            core.Input()
+        ], LibEntryComponent.prototype, "user", void 0);
+        __decorate([
+            core.Input()
+        ], LibEntryComponent.prototype, "baseUrl", void 0);
+        __decorate([
+            core.Input()
+        ], LibEntryComponent.prototype, "nlBaseUrl", void 0);
+        __decorate([
+            core.Input()
+        ], LibEntryComponent.prototype, "blobUrl", void 0);
+        __decorate([
+            core.Input()
+        ], LibEntryComponent.prototype, "botPhoneNumber", void 0);
+        __decorate([
+            core.Input()
+        ], LibEntryComponent.prototype, "resourceService", void 0);
+        __decorate([
+            core.Input()
+        ], LibEntryComponent.prototype, "adapterId", void 0);
+        __decorate([
+            core.Input()
+        ], LibEntryComponent.prototype, "broadcastAdapterId", void 0);
+        LibEntryComponent = __decorate([
+            core.Component({
+                selector: 'lib-lib-entry',
+                template: "<div class=\"ui\">\n    <div class=\"relative position\">\n        <div class=\"py-8 ua-btn-container ua-back-btn-container px-20 z-index-2\">\n           <div class=\"mx-20\">\n               <button mat-raised-button (click)=\"goBack()\">\n                   <mat-icon>keyboard_backspace</mat-icon>\n                   {{resourceService?.frmelmnts?.btn?.back || 'Back'}}\n               </button>\n           </div>\n        </div>\n    </div>\n    <div class=\"uci-admin\">\n        <div class=\"uci-admin-content\">\n            <div class=\"uci-admin-left-panel ua-side-content z-index-2\">\n                <lib-side-panel></lib-side-panel>\n            </div>\n            <div class=\"uci-admin-right-panel ua-main-content z-index-2\">\n                <router-outlet></router-outlet>\n            </div>\n        </div>\n    </div>\n</div>\n",
```

Comment:
can't we store the text of template in some string variable and assign it to the template key ,same for the styles url ?